inst/help/mlclassificationneuralnetwork.md

```diff
@@ -0,0 +1,85 @@
+Neural Network Classification
+==========================
+
+Feedforward neural networks are predictive algorithms inspired by the biological neural networks that constitute brains. A neuron (node) that receives a signal then processes it and can send signals to neurons connected to it. The signal at a node is a real number, and the output of each node is computed by sending the signal trough the activation function. The number of nodes in the network is intrinsincly linked to model complexity, as high numbers increase the flexibility of the model.
```

Comment:
The number of layers however contributes much more to model complexity compared to the number of nodes.